src/bowling.js

```diff
@@ -0,0 +1,43 @@
+class Game {
+
+  constructor() {
+    this.gameScore = 0;
+    this.frame = 1;
+    this.currentRoll = 1;
+    this.currentFrameScore = 0;
+    this.frameScores = [];
+  };
+
+roll(pins) {
```

Comment:
This function seems to be very long. Could be divided into several functions: function that throws an error, function that adds to this.frame, function that adds to this.currentFrameScore.